coalib/bearlib/aspects/Smell.py

```diff
@@ -294,24 +510,52 @@ class docs:
 @Smell.subaspect
 class Complexity:
     """
-    This aspect checks on the cyclomatic complexity of your code.
+    Complexity of a code based on different complexity metrics.
     """
     class docs:
         example = """
-        for (i=0; i<n; ++i){
```

Comment:
could you show me the review comment which said to replace a real example with a textual description without an example in this variable?